### DISCUSSION TITLE
can i use fl_query 1.0.0-alpha.3 ?

### DISCUSSION THREAD

**Discussion by affansk** [KRTirtho/fl-query#26] [Q&A]
can i use fl_query 1.0.0-alpha.3 ? is it Stable for building production level App ?

**Comment by KRTirtho** [comment]
Although alpha.3 is quite stable and is being used in production on one of my app (spotube) but I'd still not guarantee best stability
There's some other features planned that may need the API to be modified. Which can be breaking changes. So I'll suggest to wait a little longer

**Reply by affansk** [reply]
Please also we need feature like onError to catch global error, if you see Graphql and react query have this feature

**Comment by affansk** [comment]
@KRTirtho will this package be maintain in future as well ? because this package is amazing. want to take full advantage of the features in it.

**Reply by KRTirtho** [reply]
Yes it'll be maintained (bcuz I've multiple apps completely dependent on it). It's just that I'm currently busy with some personal stuff (mostly exams)

**Reply by affansk** [reply]
@KRTirtho would you mind telling me. when are you planning to release stable version of this package ?

**Reply by KRTirtho** [reply]
Next month hopefully
